server/models/List.js

```diff
@@ -0,0 +1,20 @@
+const mongoose = require('mongoose')
+
+const ListSchema = new mongoose.Schema({
+  name: {
+    type: String,
+    required: true,
+  },
+  image: {
+    type: String,
+  },
+  products: [{ type: mongoose.Schema.Types.ObjectId, ref: 'Product' }],
```

Comment:
Can you run prettier on these files just so formatting is aligned and convention is followed